assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss

```diff
@@ -286,7 +291,7 @@
     }
 
     @include mq($from: leftCol) {
-      width: 490px;
+      width: 550px;
```

Comment:
<img src="https://user-images.githubusercontent.com/3072877/45751886-a8f39d00-bc0b-11e8-89d3-64753862d474.png" width=300px>
because otherwise ⬆️

Reply:
It doesn't still say 'plus some extra features...' does it?

Reply:
Actually, I'm guessing this was an old/interim screen grab?

Reply:
Oh, this is just meant to illustrate why that width has increased. The screenshots in the table should be up to date! e.g. https://user-images.githubusercontent.com/3072877/45887552-e7c45700-bdb3-11e8-8e7b-259f672f10bc.png